js/handlers/getPosts.mjs

```diff
@@ -0,0 +1,58 @@
+import { renderPosts } from "../templates/posts.mjs";
```

Comment:
Refactor these imports. They all come from the same file so they can all use one declaration.